analyses/cell-type-wilms-tumor-06/notebook/07_combined_annotation_across_samples_exploration.Rmd

```diff
@@ -398,121 +541,499 @@ cell_type_df <- cell_type_df |>
+                   
+                   "has_dupli_chr20p",
+                   "has_dupli_chr20q",
+                   
+                   "has_dupli_chr21p",
+                   "has_dupli_chr21q",
+                   
+                   "has_dupli_chr22p",
+                   "has_dupli_chr22q"
+                   )
+
+#################################################################################
+
+# define the vector of ordered chromosomes arms##################################
+has_loss_chr <-  c("has_loss_chr1p",                                        
+                   "has_loss_chr1q",
+
+                   "has_loss_chr2p",
+                   "has_loss_chr2q",
+                   
+                   "has_loss_chr3p",
+                   "has_loss_chr3q",
+                   
+                   "has_loss_chr4p",
+                   "has_loss_chr4q",
+                   
+                   "has_loss_chr5p",
+                   "has_loss_chr5q",
+                   
+                   "has_loss_chr6p",
+                   "has_loss_chr6q",
+                   
+                   "has_loss_chr7p",
+                   "has_loss_chr7q",
+                   
+                   "has_loss_chr8p",
+                   "has_loss_chr8q",
+                   
+                   "has_loss_chr9p",
+                   "has_loss_chr9q",
+                   
+                   "has_loss_chr10p",
+                   "has_loss_chr10q",
+                   
+                   "has_loss_chr11p",
+                   "has_loss_chr11q",
+                   
+                   "has_loss_chr12p",
+                   "has_loss_chr12q",
+                   
+                   "has_loss_chr13p",
+                   "has_loss_chr13q",
+                   
+                   "has_loss_chr14p",
+                   "has_loss_chr14q",
+                   
+                   "has_loss_chr15p",
+                   "has_loss_chr15q",
+                   
+                   "has_loss_chr16p",
+                   "has_loss_chr16q",
+                   
+                   "has_loss_chr17p",
+                   "has_loss_chr17q",
+                   
+                   "has_loss_chr18p",
+                   "has_loss_chr18q",
+                   
+                   "has_loss_chr19p",
+                   "has_loss_chr19q",
+                   
+                   "has_loss_chr20p",
+                   "has_loss_chr20q",
+                   
+                   "has_loss_chr21p",
+                   "has_loss_chr21q",
+                   
+                   "has_loss_chr22p",
+                   "has_loss_chr22q"
+                   )
+
+
+#################################################################################
+# Here we define the size of each chromosome arm that will be used for plotting of the mean CNV profile
+
+  # Load cytoBand file
+  cytoBand <- read.table(gzfile(arm_order_file), header = FALSE, sep = "\t", stringsAsFactors = FALSE)
```

Comment:
Two comments here:

1. Is there a reason to use this file and not the one we created for use with CNV? Doesn't that already contain a lot of this information, and we can reduce some of the code here?
2. Even though this file isn't used until this section of the notebook, we should still read it in towards the setup chunks at the top to keep all reading in one place.